Maps_Enum_HW/homework3/task1/StudentService.java

```diff
@@ -0,0 +1,55 @@
+package homework3.task1;
+
+import java.util.List;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class StudentService {
```

Comment:
Please create these maps inside the appropriate methods, otherwise you will get a mess when calling the methods multiple times.